src/Microsoft.PowerShell.Commands.Utility/commands/utility/MatchString.cs

```diff
@@ -76,77 +73,54 @@ public object Clone()
     /// </summary>
     public class MatchInfo
     {
-        private static string s_inputStream = "InputStream";
+        private static readonly string s_inputStream = "InputStream";
 
         /// <summary>
-        /// Indicates if the match was done ignoring case.
+        /// Gets or sets a value indicating whether the match was done ignoring case.
         /// </summary>
         /// <value>True if case was ignored.</value>
         public bool IgnoreCase { get; set; }
 
         /// <summary>
-        /// Returns the number of the matching line.
+        /// Gets or sets the number of the matching line.
         /// </summary>
         /// <value>The number of the matching line.</value>
         public int LineNumber { get; set; }
 
         /// <summary>
-        /// Returns the text of the matching line.
+        /// Gets or sets the text of the matching line.
         /// </summary>
         /// <value>The text of the matching line.</value>
         public string Line { get; set; } = string.Empty;
 
-        /// <summary>
-        /// Returns the base name of the file containing the matching line.
-        /// <remarks>
-        /// It will be the string "InputStream" if the object came from the input stream.
-        /// This is a readonly property calculated from the path. <see cref="Path"/>
-        /// </remarks>
-        /// </summary>
-        /// <value>The file name</value>
-        public string Filename
```

Comment:
Please revert the remove - we can not change public contract.

Reply:
Should we change to comment to indicated that this isn't used anywhere?

Reply:
I think we shouldn't block the PR and please open new tracking Issue to get PowerShell Committee conclusion.